CHANGELOG.md

```diff
@@ -10,10 +10,12 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 ### Changed
 
+- For some data types (plist, json, yaml and core data) resource synthesizers now group them and let `SwiftGen` output a single fine instead of one for each resource.  [#2887](https://github.com/tuist/tuist/pull/2887) by [@fila95](https://github.com/fila95)
```

Comment:
this is potentially breaking if you had some custom template relying on working on a single file. I suspect there are not many of those out there, so I feel it is fine to do so, but I'd mark it as such

Reply:
Added the Breaking message 🚀